### ISSUE TITLE
spectra displayer & tracking line

### ISSUE THREAD

**Issue by NPellet** [NPellet/visualizer#935]
Hey @targos,
I implemented a basic method to allow the tracking line on individual series:
graph.getSerie("CA").allowTrackingLine( {} );
Does not need any prior options given to the graph instance. Can you implement an option in the visualizer. This way, when it breaks, I'll know soon enough :D

**Comment by targos** [comment]
Trying to implement this, I get the following error:
```
Unhandled rejection TypeError: Cannot read property 'map' of undefined
    at EventEmitter.trackingLine (http://localhost/github/visualizer/src/components/jsgraph/dist/jsgraph.js:3086:27)
    at EventEmitter.addSerieToTrackingLine (http://localhost/github/visualizer/src/components/jsgraph/dist/jsgraph.js:3121:18)
    at EventEmitter.Graph.build../series/graph.serie.Serie.allowTrackingLine (http://localhost/github/visualizer/src/components/jsgraph/dist/jsgraph.js:11658:20)
```

**Comment by NPellet** [comment]
Ah sorry, forgot to release. Try now

2016-07-11 10:36 GMT+02:00 Michaël Zasso <notifications@github.com>:

**Comment by targos** [comment]
So this is just adding a rectangle with the value of the current point ?

**Comment by targos** [comment]
I mean even if I don't call allowTrackingLine(), there is still the black moving dot

**Comment by NPellet** [comment]
Yes this is the most minimalistic way.
If you don't want common tracking, you can use graph.trackingLine( { mode:
"common", textMethod: function( data ) { return "content" }  } );

You want to keep the mode individual (each serie has its legend), then you
can use serie.allowTrackingLine( { textMethod: function( data ) { return
"content for only one serie"; } } );

2016-07-11 10:46 GMT+02:00 Michaël Zasso <notifications@github.com>:

**Comment by NPellet** [comment]
Ah but that's because you called graph.trackingLine, no ?

2016-07-11 10:48 GMT+02:00 Norman Pellet <norman.pellet@gmail.com>:

**Comment by targos** [comment]
No I never called this method. I use `options.onMouseMoveData` to enable mouse tracking

**Comment by NPellet** [comment]
Ah yes so onmousemovedata has no visual feedback. The tracking line is
exclusively for visuals. I wanted to make something similar to

http://webbook.nist.gov/cgi/cbook.cgi?ID=C7732185&Units=SI&Mask=2&Type=JANAFL&Plot=on#JANAFL


2016-07-11 10:52 GMT+02:00 Michaël Zasso <notifications@github.com>:

**Comment by targos** [comment]
implemented in d0473032350c8d3f91302d48105640fd8fdecdf4
